index.html

```diff
@@ -184,9 +183,42 @@ <h6> I can help you build a product, feature or website Look through some of my
                                 <ul>
                                 </div>
                               </div>
-                      </div>
-                          
-                  
+                            </div> 
+                            <section>
+                              <p class="about-p">
+                                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                              </p>
+                              <form action="https://formspree.io/f/xwkjrzvw"
+                              method="POST">
+                                <ul class="inputs">
+                                   <li>
+                                   <label for="name"></label>
+                                    <input type="text" id="name" name="user_name" placeholder="Full name"/>
+                                  </li>
+                                  <li>
+                                    <label for="email"></label>
```

Comment:
Please remember, your design should look almost like the original. 
- [ ] please, kindly add some padding to the input placeholders. They are too close to the borders
- [ ] please centralize the email under the button like in the original design
- [ ] kindly add the black line at the bottom like in the original design
- [ ] the border color for the name and email input is #D0D9D4. please add that

your design             |  figma design
:-------------------------:|:-------------------------:
![Screenshot (934)_LI](https://user-images.githubusercontent.com/65298048/229867669-1e917d93-6920-446a-bacd-76d0f4654892.jpg)  |  ![Screenshot (935)](https://user-images.githubusercontent.com/65298048/229867819-1c1d32d3-b34c-41a3-bb37-90ad473541ff.png)